### PR TITLE
first part of the task done

### PR DESCRIPTION
**Pull request by less08**

### CODE REVIEW

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+
+public class ConsoleHandler {
+    private final BetDao betDao = new BetDaoImpl();
+    private final UserDao userDao = new UserDaoImpl();
+
+    public void handleBet() {
+        Scanner scanner = new Scanner(System.in);
+        while (true) {
+            String command = scanner.nextLine();
+            if (command.equalsIgnoreCase("q")) {
```
Comment (vicire) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java]: make "q" constant

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+                return;
+            }
+            Bet bet = null;
+            try {
+                String[] betData = command.split(" ");
```
Comment (vicire) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java]: Same with " "

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+                int value = Integer.parseInt(betData[0]);
```
Comment (vicire) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java]: Magic number here and under

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+            betDao.addBet(bet);
+            System.out.println(bet == null ? null : bet.toString());
+        }
+    }
+
+    public void handleUser() {
+        Scanner scanner = new Scanner(System.in);
+        while (true) {
+            String command = scanner.nextLine();
+            if (command.equalsIgnoreCase("q")) {
```
Comment (vicire) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java]: Same as in handleBet

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+                return;
+            }
+            Bet bet = null;
+            try {
+                String[] betData = command.split(" ");
+                int value = Integer.parseInt(betData[0]);
+                double risk = Double.parseDouble(betData[1]);
+                bet = new Bet(value, risk);
+            } catch (NumberFormatException e) {
```
Comment (vicire) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java:38]: What happened if we enter 1 String?

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+                System.out.println("Please, enter correct data:");
+            }
+            betDao.addBet(bet);
```
Comment (vicire) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java]: Do we need to add null in storage?

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 7]
```diff
@@ -0,0 +1,55 @@
...
+                System.out.println(userDao.getAll());
+                return;
+            }
+            String[] userData = command.split(" ");
+            String name = userData[0];
+            String sport = userData[1];
+            if (name.length() == 0 || sport.length() == 0) {
+                System.out.println("Please, enter correct name and favourite sport");
+            } else {
+                User user = new User(name,sport);
```
Comment (vicire) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java]: What happened if we enter 1 or 3 Strings in command line?

**Review by vicire — CHANGES_REQUESTED** [review]

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 8]
```diff
@@ -9,26 +9,35 @@
...
             }
             Bet bet = null;
             try {
-                String[] betData = command.split(" ");
-                int value = Integer.parseInt(betData[0]);
-                double risk = Double.parseDouble(betData[1]);
+                String[] betData = command.split(DIVIDER);
+                if (betData.length != 2) {
+                    throw new RuntimeException("Incorrect data, expected pair of values");
+                }
```
Comment (Max-Pochepets) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java:33]: we could have just add another `IndexOutOfBound` exception to `catch` block

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 9]
```diff
@@ -37,17 +46,20 @@ public void handleUser() {
...
                 System.out.println(userDao.getAll());
                 return;
             }
-            String[] userData = command.split(" ");
-            String name = userData[0];
-            String sport = userData[1];
+            String[] userData = command.split(DIVIDER);
+            if (userData.length != 2) {
+                throw new RuntimeException("Incorrect data, expected pair of values");
+            }
```
Comment (Max-Pochepets) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java:56]: same here

**Review by Max-Pochepets — APPROVED** [review]

File: src/main/java/core/basesyntax/controller/ConsoleHandler.java [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+                    throw new RuntimeException("Incorrect data, expected pair of values");
+                }
+                int value = Integer.parseInt(betData[INDEX_VALUE]);
+                double risk = Double.parseDouble(betData[INDEX_RISK]);
+                bet = new Bet(value, risk);
+                betDao.addBet(bet);
+            } catch (NumberFormatException e) {
+                System.out.println("Please, enter correct data:");
+            }
+            System.out.println(bet == null ? null : bet.toString());
```
Comment (RomanDubovskyi) [inline src/main/java/core/basesyntax/controller/ConsoleHandler.java:41]:
```suggestion
            System.out.println(bet);
```

**Review by RomanDubovskyi — APPROVED** [review]

**Pull request merged**